ulog/log.go

```diff
@@ -153,12 +155,17 @@ func (l *baseLogger) Log(lvl zap.Level, message string, keyVals ...interface{})
 	}
 }
 
+type stackTracer interface {
```

Comment:
Nvm, Cause is just for the original error, if you need that sort of thing, which in here we dont.